articles/tensorflow-vs-pytorch/index.md

```diff
@@ -0,0 +1,113 @@
+5. It supports Nvidia's [CUDA](https://en.wikipedia.org/wiki/CUDA), ensuring that the code runs on a GPU, decreasing the time needed to run the code.
+6. Dynamic Computation Graphs is a key feature in PyTorch. It ensures that the graphs build up dynamically as you code.
+
+Let's begin by discussing some of the comparisons between TensorFlow and PyTorch. 
+
+### Computation Graphs
+
+From the start, the computation graph was a notable difference between TensorFlow and PyTorch.
+
+From its onset, PyTorch was built around the concept of dynamism. It provided developers with the ability to dynamically define and manipulate graphs at runtime while it is executing. This feature offered PyTorch developers an advantage over TensorFlow as manipulating graphs during runtime helped easily troubleshoot the model in case of any issue with the code.
+
+By default, when [TensorFlow](https://www.tensorflow.org/) was developed, it let developers create static computation graphs at compile time. It didn't allow us to define and manipulate graphs at runtime. If we wanted to make any changes, we would have to rebuild the neural network from the start. 
+
+But, with the release of TensorFlow 2.0, they introduced the "eager mode." They aimed to allow graphs to be defined and manipulated dynamically in computation while executing as was with PyTorch, which already had this feature, which worked so well. Rather than waiting until everything is coded up and connected before you can see if anything is done right, you could now "eagerly" execute your operations as you're working. 
+
+Today, both frameworks allow both the static and dynamic computation of graphs. No much difference between the two as far as computation graphs are concerned. 
+
+### Visualization
+
+Besides Computation Graphs, the visualization tools used between these two frameworks show notable distinctions.
+
+TensorFlow comes on-board with the [TensorBoard](https://www.tensorflow.org/tensorboard) visualization toolkit that helps researchers understand graphs and how their code runs. It is also an easy-to-use toolkit that helps track and visualize metrics such as accuracy and losses in your model. 
+
+PyTorch does not have a native in-build visualizing tool like TensorFlow does. A researcher has to import external visualization libraries such as [matplotlib](https://matplotlib.org/) and [seaborn](https://seaborn.pydata.org/) to be able to achieve the same objective.
+
+They both achieve the same visualization objective, but TensorBoard offers researchers a bigger advantage. It has more features and a good number of visualization tools that transform graphs into beautiful-looking visualizations.
+
+### Community
+
+As I mentioned earlier, TensorFlow has been around for a longer time compared to PyTorch. Due to this factor, and partly because Google's technology giants develop it, TensorFlow has a much wider community around its ecosystem. This community helps make the learning of the framework easier.
```

Comment:
This statement is actually wrong. The research community has embraced PyTorch really well. Therefore, I suggest you do some research and update this statement. PyTorch has a solid community presence.